core/client/src/main/java/alluxio/client/file/options/DeleteOptions.java

```diff
@@ -24,6 +25,7 @@
 @NotThreadSafe
 public final class DeleteOptions {
   private boolean mRecursive;
+  private boolean mRemoveUFSFile;
```

Comment:
ok, I think I understand now ... the option would allow us to remove the file from Alluxio without removing it from UFS so that you can reload it.

I would prefer to call it `mAlluxioOnly` and change its default to be `false`